samples/app/src/androidTest/java/com/sourcepoint/app/v6/MainActivityKotlinTest.kt

```diff
@@ -1187,16 +1187,15 @@ class MainActivityKotlinTest {
 
         scenario = launchActivity()
 
-        wr(backup = { clickOnRefreshBtnActivity() }) { tapAcceptAllOnWebView() }
-        wr { tapAcceptAllOnWebView() }
-
         wr { verify(exactly = 0) { spClient.onError(any()) } }
         wr { verify(exactly = 1) { spClient.onSpFinished(any()) } }
 
         // TODO check if the applies value of GDPR changed to TRUE
         scenario.onActivity { activity ->
```

Comment:
you need to wrap this condition into a wr{} otherwise the test will skip it

Reply:
> you need to wrap this condition into a wr{} otherwise the test will skip it

why would the test skip it?

I thought that `wr { ... }` is a wrapper that checks whether or not the code block inside it fails (there is a `try/catch` block inside), we also have plenty of tests that are not using that wrapper and they all seems to work fine

Reply:
right usually there are assertions that check some parameters

Reply:
@carmelo-iriti so if there is a check, we don't need a `wr { ... }` wrapping those lines, right?